Source/WebCore/dom/Document.cpp

```diff
@@ -2097,6 +2099,54 @@ std::optional<BoundaryPoint> Document::caretPositionFromPoint(const LayoutPoint&
     return { { node.releaseNonNull(), offset } };
 }
 
+RefPtr<CaretPosition> Document::caretPositionFromPoint(double x, double y, CaretPositionFromPointOptions options)
+{
+    if (!hasLivingRenderTree())
+        return nullptr;
+
+    LayoutPoint clientPoint = { x, y };
```

Comment:
Not sure we need the `=` here.

Reply:
Yes, wasteful, in fact we do not need a named variable at all (originally I was probably thinking it would be used in more than one place).